site/content/docs/5.0/components/alerts.md

```diff
@@ -128,7 +128,7 @@ This makes an alert listen for click events on descendant elements which have th
         <code>dispose</code>
       </td>
       <td>
-        Destroys an element's alert.
+        Destroys an element's alert. (Removes stored data on the DOM element and unsets the alert's attributes)
```

Comment:
We do not unsets the alert's attribute, we just remove the stored data on the DOM element that's all. https://github.com/twbs/bootstrap/blob/master/js/src/alert.js#L78-L81

Reply:
@Johann-S Updated.